documentation/manifest.md

```diff
@@ -93,9 +93,9 @@
         - [ ] Verify Issues button navigates to Issues page on GitHub.com (when logged in and the repository is enabled)
 - [ ] **Creating Multiple Repositories**
     - [ ] Go to Team Explorer Connect page (logged in)
-    - [ ] Create additional repository
-    - [ ] Publish new repository
-    - [ ] Verify remote information present in repository settings
+      - [ ] Create additional repository
+        - [ ] Publish new repository
+          - [ ] Verify remote information present in repository settings
```

Comment:
I had an edit tab open for the manifest from a few days ago, apparently I cared about the indents then. 😅 

![alt text](https://media.licdn.com/mpr/mpr/AAEAAQAAAAAAAAbXAAAAJGFmZmM1N2IzLTE2ZTQtNDZiYi1hZjY4LWUxZjViN2MwMDA2Mw.jpg)